examples/images_contours_and_fields/barb_demo.py

```diff
@@ -22,24 +22,21 @@
 data = np.array(data, dtype=[('x', np.float32), ('y', np.float32),
                              ('u', np.float32), ('v', np.float32)])
 
+fig, axes1 = plt.subplots(nrows=2, ncols=2)
```

Comment:
Since `fig`, doesn't have a number, `axes1` shouldn't either.

I propose to decide on one canonical way of writing the second argument: `axs` vs. `axes`. Both seem to be used. I slightly tend to `axs` in the sense of multiple `ax`. Since Axes and Axis are both class names `axes` may just too easily be misinterpreted as multiple `Axis` or a single `Axes` instance.

So proposed value here is `axs`.